dashboard/app/helpers/levels_helper.rb

```diff
@@ -132,9 +132,23 @@ def app_options
     view_options(is_channel_backed: true) if @level.channel_backed?
 
     return blockly_options if @level.is_a? Blockly
+    return dsl_defined_options if @level.is_a? DSLDefined
     Hash[view_options.map{|key, value|[key.to_s.camelize(:lower), value]}]
```

Comment:
This is dave replying by email on which I am on the to: line, to see if I
show up as 'dave' or 'deploy-dashboard'

On Wed, Aug 12, 2015 at 11:00 AM, Laurel <notifications@github.com> wrote:

> In dashboard/app/helpers/levels_helper.rb
> <https://github.com/code-dot-org/code-dot-org/pull/3498#discussion_r36892411>
> :
>
> > @@ -132,9 +132,23 @@ def app_options
> >      view_options(is_channel_backed: true) if @level.channel_backed?
> >
> >      return blockly_options if @level.is_a? Blockly
> > +    return dsl_defined_options if @level.is_a? DSLDefined
> >      Hash[view_options.map{|key, value|[key.to_s.camelize(:lower), value]}]
>
> This is laurel btw. Somehow I am deploy-dashboard when I comment by email??
> … <#14f231815b8125b0_>
> On Aug 12, 2015, at 10:58, deploy-dashboard-code-org <
> notifications@github.com> wrote: In
> dashboard/app/helpers/levels_helper.rb: > @@ -132,9 +132,23 @@ def
> app_options > view_options(is_channel_backed: true) if
> @level.channel_backed? > > return blockly_options if @level.is_a? Blockly >
> + return dsl_defined_options if @level.is_a? DSLDefined >
> Hash[view_options.map{|key, value|[key.to_s.camelize(:lower), value]}] Yep,
> might be clearer to change that to an plainer if instead of a series of
> fallthroughs … — Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/3498/files#r36892411>.
>